diff --git a/odoo/addons/base/__manifest__.py b/odoo/addons/base/__manifest__.py
index 043dd3ce18c..a8329b367dd 100644
--- a/odoo/addons/base/__manifest__.py
+++ b/odoo/addons/base/__manifest__.py
@@ -51,7 +51,7 @@ The kernel of Odoo, needed for all installation.
         'views/ir_qweb_widget_templates.xml',
         'views/ir_module_views.xml',
         'data/ir_module_category_data.xml',
-        'data/ir_module_module.xml',
+        # 'data/ir_module_module.xml',
         'report/ir_module_reports.xml',
         'report/ir_module_report_templates.xml',
         'wizard/base_module_update_views.xml',
